app/matplotlib.ipynb

```diff
@@ -233,22 +232,30 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 11,
-   "outputs": [],
    "source": [
     "x = np.linspace(0, 5, 100)\n",
     "y1 = np.sin(x)\n",
     "y2 = np.cos(x)\n",
     "\n",
-    "# write your code here\n"
+    "fig, axs = plt.subplots(1, 2)\n",
+    "\n",
+    "axs[0].plot(x, y1)\n",
+    "axs[0].set_title(\"Title of 1 plot\")\n",
+    "axs[0].set_xlabel(\"x label\")\n",
+    "axs[0].set_ylabel(\"y label\")\n",
+    "axs[0].set_facecolor(\"blue\")\n",
+    "\n",
+    "axs[1].plot(x, y2)\n",
+    "axs[1].set_title(\"Title of 2 plot\")\n",
+    "axs[0].set_xlabel(\"x label\")\n",
+    "axs[0].set_ylabel(\"y label\")\n",
+    "axs[1].set_facecolor(\"red\")"
```

Comment:
The task requires showing the subplots horizontally. Ensure that the `plt.subplots` call uses `nrows=1` and `ncols=2` to display them side by side.